roles/openshift_node_upgrade/tasks/rpm_upgrade.yml

```diff
@@ -12,3 +12,8 @@
 - name: Ensure python-yaml present for config upgrade
   package: name=PyYAML state=present
   when: not openshift.common.is_atomic | bool
+
+- name: Install Node service file
+  template:
+    dest: "/etc/systemd/system/{{ openshift.common.service_type }}-node.service"
+    src: "{{ openshift.common.service_type }}-node.service.j2"
```

Comment:
@sdodson Looking more at ``roles/openshift_node_upgrade/tasks/systemd_units.yml`` it seems like it's included as part of containerized upgrades only. It also includes ``docker`` commands for pulling containerized containers. It's possible we may want to rename the ``roles/openshift_node_upgrade/tasks/systemd_units.yml`` in the future, but I think it may not be the right place based on tasks.